state-chain/pallets/cf-trading-strategy/src/lib.rs

```diff
@@ -52,11 +52,79 @@ const STRATEGY_ORDER_ID: OrderId = 0;
 
 impl_pallet_safe_mode!(PalletSafeMode; strategy_updates_enabled, strategy_closure_enabled, strategy_execution_enabled);
 
+pub struct DefaultLimitOrderUpdateThresholds<T> {
+	_phantom: PhantomData<T>,
+}
+impl<T: Config> Get<BTreeMap<Asset, AssetAmount>> for DefaultLimitOrderUpdateThresholds<T> {
+	fn get() -> BTreeMap<Asset, AssetAmount> {
+		Asset::all()
+			.map(|asset| {
+				(
+					asset,
+					match asset {
+						Asset::Usdc | Asset::Usdt | Asset::ArbUsdc | Asset::SolUsdc =>
+							1_000_000_000, // $1,000 USD
+						_ => u128::MAX, // Turned off by default all other assets
```

Comment:
Same comment: we don't need to duplicate the default behaviour in our default config.